CONVENTIONS.md

```diff
@@ -264,6 +264,11 @@ Please note that in case of [Single Node OpenShift](https://github.com/openshift
   * Two replicas
   * Hard pod [anti-affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#always-co-located-in-the-same-node) on hostname (no two pods on the same node)
   * Use the maxUnavailable rollout strategy on deployments (prefer 25% by default for a value)
+  * If the operator or operand requires persistent storage
+    * Operators should keep their state entirely in CustomResources, using PV's is discouraged.
+    * If and how persistent storage impacts high availability of an operand depends on the cluster configuration, the deployment resource used (e.g. Deployment vs StatefulSet) and available storage classes, e.g. whether [topology-aware volume provisioning](https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode) is available
+    * Operands that are deployed as a StatefulSet, i.e. where replicas get a separate volume instead of all replicas sharing a volume, typically require code to sync state between replicas. THis can cause significant cross-zone traffic.
```

Comment:
```suggestion
    * Operands that are deployed as a StatefulSet, i.e. where replicas get a separate volume instead of all replicas sharing a volume, typically require code to sync state between replicas causing cross-zone traffic which may be charged.
```
wouldn't the same warning apply when multiple replicas in different AZs read from a single volume?

Reply:
If two replicas can mount the same volume, they are both in the AZ of the volume iiuc.